### PR TITLE
Added version banner to layout.

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) adds an info box to the top of the page for v2.2. This informs them that they are not on the docs for the latest version of Magento.

Fixes #1234 

## Affected DevDocs pages

- All

### CODE REVIEW

**Comment by jeff-matthews** [comment]
Fixes #1234

File: common/css/devdocs.css [hunk 1]
```diff
@@ -19,4 +20,7 @@
 
 .ref-list dt{
   font-weight:500;
+}
+.versions {
```
Comment (belbiy) [inline common/css/devdocs.css]: Ideally, i would recommend to have more descriptive class name (`versions` is too generic).

Comment (belbiy) [reply common/css/devdocs.css]: You can be more specific by using this selector:
```
.message-banner.versions
```

File: _includes/layout/page-header.html [hunk 2]
```diff
@@ -1,12 +1,19 @@
 <!-- page-header -->
 <section class="page-intro">
-	{% if page.ee_only == true %}
-		<span class="ee-only">{{site.data.var.ee}} only</span>
-	{% endif %}
+{% if page.guide_version != site.version %}
+<div class="message-banner versions">
```
Comment (belbiy) [inline _includes/layout/page-header.html]: Ideally, html should be indented for easier readability of the nested elements.

File: common/css/devdocs.css (hunk 1 above)
Comment (dobooth) [reply common/css/devdocs.css]: Done.

File: _includes/layout/page-header.html (hunk 2 above)
Comment (dobooth) [reply _includes/layout/page-header.html]: Fixed. Had to get rid of tabs.

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**